Composer/packages/client/src/components/ProjectTree/ProjectTreeHeader.tsx

```diff
@@ -3,27 +3,44 @@
 
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core';
-import { FontSizes, NeutralColors } from '@uifabric/fluent-theme';
+import { useEffect, useRef, useState } from 'react';
+import { FontSizes } from '@uifabric/fluent-theme';
 import formatMessage from 'format-message';
-import { IconButton } from 'office-ui-fabric-react/lib/Button';
-import { OverflowSet } from 'office-ui-fabric-react/lib/OverflowSet';
-import { TooltipHost, DirectionalHint } from 'office-ui-fabric-react/lib/Tooltip';
+import { CommandButton } from 'office-ui-fabric-react/lib/Button';
+import { IOverflowSetItemProps, OverflowSet } from 'office-ui-fabric-react/lib/OverflowSet';
 import { IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { ISearchBox, ISearchBoxStyles, SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 
-import { moreButton, overflowSet } from './treeItem';
+const searchBox: ISearchBoxStyles = {
+  root: {
+    borderBottom: '1px solid #edebe9',
```

Comment:
same here for the color
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/neutrals
gray30